astroid/nodes/scoped_nodes/scoped_nodes.py

```diff
@@ -2297,6 +2297,24 @@ def _newstyle_impl(self, context=None):
         doc=("Whether this is a new style class or not\n\n" ":type: bool or None"),
     )
 
+    @decorators_mod.cachedproperty
+    def fromlineno(self):
```

Comment:
```suggestion
    def fromlineno(self) -> Optional[int]:
```

Right? We could even remove the typing in the docstring?